pkg/utils/maestro/maestro.go

```diff
@@ -0,0 +1,35 @@
+package maestro
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift-online/maestro/pkg/api"
+	"github.com/openshift-online/maestro/pkg/services"
+)
+
+// FindConsumerByName checks if a consumer with the given name exists in the maestro service.
+func FindConsumerByName(ctx context.Context, consumerService services.ConsumerService, consumerName string) (bool, error) {
+	consumers, svcErr := consumerService.FindByNames(ctx, []string{consumerName})
```

Comment:
why cannot we use https://github.com/stolostron/maestro-addon/blob/973d196f7bd34ebc89d707bbe42dcbfa89cc7f84/pkg/helpers/maestro.go#L32C18-L32C24?